### ISSUE TITLE
Add support for webpack 5

### ISSUE THREAD

**Issue by kaiyoma** [jantimon/html-webpack-harddisk-plugin#36]
Currently getting this warning from yarn:

```
warning " > html-webpack-harddisk-plugin@1.0.2" has incorrect peer dependency "webpack@^1.0.0 || ^2.0.0 || ^3.0.0 || ^4.0.0".
```

**Comment by jantimon** [comment]
Can you please create a pull request? :)

Please note that webpack 5 has currently a bug with the child compiler and is therefore not ready to be used with html-webpack-plugin by now.

See also https://github.com/jantimon/html-webpack-plugin/issues/1527

**Comment by pkat** [comment]
Hi, now that the issue referenced above has been resolved, will "html-webpack-harddisk-plugin" be updated to support webpack 5?

**Comment by jantimon** [comment]
released as `html-webpack-harddisk-plugin@2.0`
